app/views/shut_mucr_summary.scala.html

```diff
@@ -41,7 +41,10 @@ <h2 id="shutMucr-header">@messages("shutMucr.summary.header")</h2>
                     <td id="shutMucr-type">@messages("shutMucr.summary.type")</td>
                     <td id="shutMucr-mucr">@mucr</td>
                     <td class="text-right">
-                        <a id="shutMucr-change" href="@routes.ShutMucrController.displayPage()">@messages("site.edit")</a>
+                        <a id="shutMucr-change" href="@routes.ShutMucrController.displayPage()">
+                            <span aria-hidden="true">@messages("site.edit")</span>
+                            <span class="visuallyhidden">@{messages("site.change.hint.disassociate.ucr")}</span>
```

Comment:
Not sure, but I think it should be `site.change.hint.shut.mucr`

Reply:
Correct. Copy paste error.

Reply:
Fixed